<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>votanteWin</class>
 <widget class="QWidget" name="votanteWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/img/votante.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerResultados">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>550</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #B7387A;
color:white;
border-radius: 15px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>VER RESULTADOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerrarSesion">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #921B63;
color:white;
border-radius: 9px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>CERRAR SESIÓN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Isaac Ruiz Vázquez</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Elecciones en curso:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resultados de Elecciones Pasadas:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblAbiertas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tblResultados">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>421</width>
     <height>411</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnParticipar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #B7387A;
color:white;
border-radius: 15px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>PARTICIPAR</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblEspera">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Elecciones en espera de resultados:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: none;
color: #B7387A;
border-radius: 15px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
